File: TaskManager/TaskManager/TaskManager/Models/Comment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TaskManager.Models
{
    using System;

    namespace TaskManager.Models
    {
        public class Comment
        {
            public Guid Id { get; set; }
            public string Content { get; set; }
            public string Author { get; set; }
            public DateTime CreatedAt { get; set; }

            public Comment()
            {
                Id = Guid.NewGuid();
                CreatedAt = DateTime.UtcNow;
            }
        }
    }
}

File: Task Manager/src/Comment.java
import java.util.Date;

public class Comment {
    private String id;
    private String content;
    private String author;
    private Date createdAt;

    public Comment(String id, String content, String author) {
        this.id = id;
        this.content = content;
        this.author = author;
        this.createdAt = new Date();
    }

}
